lib/google_directions_api/directions.rb

```diff
@@ -55,43 +55,31 @@ def data
     end
 
     def total_distance
-      if waypoints_present?
-        meters = 0
-        data["routes"][0]["legs"].each do |leg|
-          meters += leg["distance"]["value"]
-        end
-        return meters
-      else
-        data["routes"][0]["legs"][0]["distance"]["value"]
+      meters = 0
+      data["routes"][0]["legs"].each do |leg|
+        meters += leg["distance"]["value"]
       end
+
+      meters
```

Comment:
Fill like there is some creativity that could be done with the inject method to cut down on the number of lines

Reply:
Agree, done